### PR TITLE
Issue 27 improve es5 performance

### PR DESCRIPTION
**Pull request by omittones**
Improve performance and size when compiling for ES5. Will cause things to break if using classes of endpoints and/or classes of data types. Had to convert everything to interfaces. #27

### CODE REVIEW

File: Watts.nuspec [hunk 1]
```diff
@@ -2,15 +2,15 @@
...
+    <version>2.0.0</version>
     <authors>Balakrishnan (Balki) Ranganathan</authors>
     <owners>Balakrishnan (Balki) Ranganathan</owners>
     <licenseUrl>https://raw.githubusercontent.com/greymind/WebApiToTypeScript/master/LICENSE</licenseUrl>
     <projectUrl>https://github.com/greymind/WebApiToTypeScript</projectUrl>
     <requireLicenseAcceptance>false</requireLicenseAcceptance>
     <description>A tool for code generating TypeScript endpoints, interfaces, enums, resources and view paths for your ASP.NET Web API controllers/web apps</description>
     <summary>A tool for code generating TypeScript endpoints, interfaces, enums, resources and view paths for your ASP.NET Web API controllers/web apps</summary>
-    <releaseNotes>Fix getting base types in certain cases</releaseNotes>
+    <releaseNotes>Optimize code for execution and size when compiling TypeScript to ES5.</releaseNotes>
```
Comment (greymind) [inline Watts.nuspec]: Can you please add a note that the major breaking change was the removal of classes from interface.ts? cc @dborovic

File: src/Watts/Program.cs [hunk 2]
```diff
@@ -1,27 +1,41 @@
...
             {
-                Console.WriteLine("Usage: Watts.exe <\"Path/To/Config.json\">");
-                return 1;
+                var path = Path.Combine(Environment.CurrentDirectory, "watts.config.json");
+                if (File.Exists(path))
+                    watts.ConfigFilePath = path;
             }
-
-            var watts = new WebApiToTypeScript.WebApiToTypeScript
+            else if (args.Length > 0 && File.Exists(args[0]))
```
Comment (greymind) [inline src/Watts/Program.cs]: Shouldn't this be File.Exists(args[1]) 1 instead of 0?

File: Watts.nuspec (hunk 1 above)
Comment (greymind) [inline Watts.nuspec]: Oh just noticed you added a config variable. In any case, can you please add what the breaking changes we need to look out for? We should also start a section on readme

Comment (omittones) [inline Watts.nuspec]: Done.

File: src/Watts/Program.cs (hunk 2 above)
Comment (omittones) [inline src/Watts/Program.cs]: The other one should be 0. Done.

**Pull request merged**